### ISSUE TITLE
Puppet-lint 2.4.0 - ERROR: Syntax error on line x

### ISSUE THREAD

**Issue by dervolker** [rodjek/puppet-lint#887]
Hello, 

Since yesterday we suddenly get an **ERROR: Syntax error on line** error. Before, no error was shown.

It seems to appear in file definitions where  an `inline_epp` is used like in the following snippet.

```
  -> file { '/etc/security/limits.d/artifactory.conf':
    content => inline_epp(@(EOT)
artifactory soft nofile <%= $roles_repo_servers::max_files %>
artifactory hard nofile <%= $roles_repo_servers::max_files %>
artifactory soft nproc <%= $roles_repo_servers::max_procs %>
artifactory hard nproc <%= $roles_repo_servers::max_procs %>
@EOT
    )
}
```
